tests/test_qeventloop.py

```diff
@@ -10,12 +10,17 @@
 from concurrent.futures import ThreadPoolExecutor, ProcessPoolExecutor
 import socket
 import subprocess
+import weakref
+import gc
 
 import quamash
 
 import pytest
 
 
+from PyQt5.QtCore import QObject, pyqtSignal
```

Comment:
can't have a hard dep on PyQt5. I might drop support for Qt4 soon though (for simplicity).

Reply:
Oh, sorry for that! I must've missed that when rebasing. I'll fix it right away.